### ISSUE TITLE
AndroidManifest not set up for PhoneGap with SDK Tools rev. 14 and later

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#45]
```
SDK 14 changed the format of the created AndroidManifest.xml.

The PhoneGap wizard is not making the correct adjustments for PhoneGap

Workaround:

Use phonegap-1.1.0/Android/Sample/AndroidManifest.xml to update 
supports-screens, uses-permissions, debuggable, 
android:configChanges="orientation|keyboardHidden", and the DroidGap activity.

```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 30 Oct 2011 at 10:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 30 Oct 2011 at 10:48
- Changed title: **AndroidManifest not set up for PhoneGap with SDK Tools rev. 14 and later**

**Comment by GoogleCodeExporter** [comment]
```
Fixed and released in 1.2.5
```

Original comment by `paul.beu...@gmail.com` on 1 Nov 2011 at 8:39
- Changed state: **Fixed**
